pkgs/by-name/du/duckstation/package.nix

```diff
@@ -1,35 +1,43 @@
-{ lib
-, stdenv
-, fetchFromGitHub
-, SDL2
-, cmake
-, cubeb
-, curl
-, extra-cmake-modules
-, libXrandr
-, libbacktrace
-, makeWrapper
-, ninja
-, pkg-config
-, qtbase
-, qtsvg
-, qttools
-, qtwayland
-, substituteAll
-, vulkan-loader
-, wayland
-, wrapQtAppsHook
```

Comment:
Personally, I think this is not a required change, and it enlarges the diff for a weak reason, but it is your right to format the expression how you wish, since you are a maintainer.